gef-ui/server/controllers/application-details/index.js

```diff
@@ -347,10 +346,68 @@ const applicationDetails = async (req, res, next) => {
       params.link += '/unissued-facilities';
     }
 
-    const amendmentsInProgressOnDeal = await getPortalAmendmentsOnDeal({ dealId, statuses: PORTAL_AMENDMENT_INPROGRESS_STATUSES, userToken });
+    const amendmentsUnderwayOnDeal = await getPortalAmendmentsOnDeal({ dealId, statuses: PORTAL_AMENDMENT_INPROGRESS_STATUSES, userToken });
 
-    params.canIssuedFacilitiesBeAmended =
-      canUserAmendIssuedFacilities(application.submissionType, application.status, userRoles) && !amendmentsInProgressOnDeal.length;
+    // array of facility ids that are currently being amended
+    const amendmentsInProgress = amendmentsUnderwayOnDeal.map((amendment) => ({
+      amendmentId: amendment.amendmentId,
```

Comment:
```suggestion
      amendmentId,
      facilityId,
      status,
```

Keep variables clean please.

Reply:
done